src/main/kotlin/eu/europa/ec/eudi/sdjwt/JwtBase64.kt

```diff
@@ -15,17 +15,23 @@
  */
 package eu.europa.ec.eudi.sdjwt
 
-import kotlin.io.encoding.Base64
-import kotlin.io.encoding.ExperimentalEncodingApi
+import java.util.*
+import java.util.Base64.Decoder
+import java.util.Base64.Encoder
 
-@OptIn(ExperimentalEncodingApi::class)
 object JwtBase64 {
 
-    fun decode(value: String): ByteArray = Base64.UrlSafe.decode(value)
+    /**
+     * Base64 encoder that doesn't add padding.
+     */
+    private val encoder: Encoder = Base64.getUrlEncoder().withoutPadding()
 
-    // Since the complement character "=" is optional,
-    // we can remove it to save some bits in the HTTP header
-    fun encode(value: ByteArray): String = removePadding(Base64.UrlSafe.encode(value))
+    /**
+     * Base64 decoder that doesn't require padding.
+     */
+    private val decoder: Decoder = Base64.getUrlDecoder()
 
+    fun decode(value: String): ByteArray = decoder.decode(value)
+    fun encode(value: ByteArray): String = encoder.encodeToString(value)
     fun removePadding(value: String): String = value.replace("=", "")
```

Comment:
Can you think an alternative that doesn't expose the removePadding?

Reply:
In the docs of `SdJwtDigest` we state that it represents a Base64 UrlSafe encoded value without padding. To me, it makes sense to move the responsibility of removing the padding to `SdJwtDigest` and keep `JwtBase64` only for encoding/decoding purposes.

ie.
```kotlin
package eu.europa.ec.eudi.sdjwt

import java.security.MessageDigest

/**
 * The digest of a [presentation][SdJwt.Presentation].
 * It contains the base64-url encoded digest of a presentation with all padding characters removed.
 */
@JvmInline
value class SdJwtDigest private constructor(val value: String) {

    companion object {
        const val CLAIM_NAME = "sd_hash"

        fun wrap(value: String): Result<SdJwtDigest> = runCatching {

//
//Remove padding below
//
            val clean = value.removePadding()
            JwtBase64.decode(clean)
            SdJwtDigest(clean)
        }

        fun digest(hashAlgorithm: HashAlgorithm, value: String): Result<SdJwtDigest> = runCatching {
            require(value.contains("~"))
            fun String.noKeyBinding() =
                if (endsWith("~")) {
                    this
                } else {
                    removeRange(lastIndexOf("~") + 1, length)
                }

            val digestAlgorithm = MessageDigest.getInstance(hashAlgorithm.alias.uppercase())
            val digest = digestAlgorithm.digest(value.noKeyBinding().encodeToByteArray())

//
//Remove padding below
//
            SdJwtDigest(JwtBase64.encode(digest).removePadding())
        }
    }
}

private fun String.removePadding() = replace("=", "")

```

Reply:
Can we find in the SD-JWT specification any requirements about padding for disclosures?